defaults/main.yml

```diff
@@ -17,3 +17,5 @@ php_xdebug_idekey: sublime.xdebug
 
 php_xdebug_max_nesting_level: 256
 
+# Only used on Debian/Ubuntu.
+php_xdebug_cli_disable: no
```

Comment:
I don't understand how this would be possible on CentOS.

I've never worked with CentOS before, but after a brief investigation it appears that PHP config files can't distinguish between different execution contexts (CLI vs FPM) like they can on Ubuntu. You'd have actually pass a unique CLI-specific configuration file as an argument to `php`, which seems like a pretty big change just to support this one use case.

A brief Google seems to confirm this: https://gagor.pl/2014/05/changing-default-php-ini-file-for-php-cli-on-centos/